src/nvim/api/private/helpers_lua.c

```diff
@@ -0,0 +1,82 @@
+// This is an open source non-commercial project. Dear PVS-Studio, please check
```

Comment:
this filename is a little weird. The long term goal is to get rid of the arbitrary namespace of "helper" functions (like misc1/misc2.c all over again) and instead group utility functions together with the API functions that use them. Nothing else changed, these functions would now belong in `api/buffer.c` which is still a bit big, but no longer painfully so.

Reply:
Agree. I hated that this file too. All done.